### ISSUE TITLE
Reward to Change class works only once. 

### ISSUE THREAD

**Issue by sharkybushbaby** [FlyingPikachu/Quests#208]
Ive created a Npc for Each class (role) on the server. So if a player wants to change Role at anytime they can, by doing a small quest. 

The Reward Command is set as  e.g "pexrank <player> Mystic"
It worked the first time but after  when reloading (questadmin reload) the command changes to "pexrank  Mystic"  (no <player>) and the reward is no longer given. 

can I get some help . Ive been losing it over this one. 
Also when going back to Edit some Class/group/role in edit quest. I literally get disconnected from the server.

**Comment by FlyingPikachu** [comment]
Sounds like you're trying to give the Mystic rank to someone whose already a Mystic. That would not be a Quests issue. If I am misunderstanding and this is still a problem, please create a new Issue.
